### PR TITLE
just sum smol improvements

### PR DESCRIPTION
**Pull request by TopiSenpai**

### CODE REVIEW

File: api/command.go [hunk 1]
```diff
@@ -30,6 +30,16 @@ func (c Command) FromGuild() bool {
 	return c.GuildID == nil
 }
 
+// ToCreate return the CommandCreate for this Command
+func (c *Command) ToCreate() *CommandCreate {
```
Comment (EveCodes31) [inline api/command.go:34]: What exactly is this for?

Comment (TopiSenpai) [reply api/command.go:34]: I actually use it here <https://github.com/DisgoOrg/disgommand/blob/5b1a6023218f3fe946bdb3628514f28c07249a24/command.go#L29>

Comment (EveCodes31) [reply api/command.go:34]: Ah, for getting the struct ready to create the command 👍

File: api/restclient.go [hunk 2]
```diff
@@ -6,14 +6,8 @@ import (
...
-// Errors when connecting to discord
-var (
-	ErrBadGateway                 = errors.New("bad gateway could not reach discord")
-	ErrUnauthorized               = errors.New("not authorized for this endpoint")
-	ErrBadRequest                 = errors.New("bad request please check your request")
-	ErrRatelimited                = errors.New("too many requests")
-	ErrTooMuchCommands = errors.New("you can provide a max of 100 application commands")
-)
+// ErrTooMuchCommands returned if a Guild reached max of 100 Command(s)
+var ErrTooMuchCommands = errors.New("you can provide a max of 100 application commands")
```
Comment (EveCodes31) [inline api/restclient.go:10]: This should be ErrToManyCommands

**Review by EveCodes31 — CHANGES_REQUESTED** [review]

File: api/restclient.go (hunk 2 above)
Comment (TopiSenpai) [reply api/restclient.go]: ErrMaxCommands would be even better I guess

Comment (EveCodes31) [reply api/restclient.go]: That works

**Review by EveCodes31 — APPROVED** [review]

**Pull request merged**